src/simulator/world.py

```diff
@@ -135,8 +135,9 @@ def run(self) -> None:
             self.__num_evolutions += 1
 
         self.dump_all_shapes_info()
+        self.dumb_all_owners_inf()
         self.dump_vis_data_to_file()
-
+        
```

Comment:
Please review all your changes before you commit your code. All changes should be meaningful. This change is not meaningful.